<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorldEventDialogWidget</class>
 <widget class="QWidget" name="WorldEventDialogWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>614</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>World Event</string>
  </property>
  <layout class="QVBoxLayout" name="rootLayout">
   <property name="spacing">
    <number>6</number>
   </property>
   <property name="leftMargin">
    <number>8</number>
   </property>
   <property name="topMargin">
    <number>8</number>
   </property>
   <property name="rightMargin">
    <number>8</number>
   </property>
   <property name="bottomMargin">
    <number>8</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="headerLayout">
     <item>
      <widget class="QLabel" name="eventIconPlaceholder">
       <property name="minimumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::Box</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Sunken</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignmentFlag::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="headerForm">
       <item row="2" column="0">
        <widget class="QLabel" name="placementLabel">
         <property name="visible">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Placement</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="eventNameLabel">
         <property name="text">
          <string>Event Name</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="eventNameLineEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="stringsCountLabel">
         <property name="visible">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>String Slots</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QWidget" name="placementWidget" native="true">
         <property name="visible">
          <bool>false</bool>
         </property>
         <layout class="QHBoxLayout" name="placementLayout">
          <property name="spacing">
           <number>4</number>
          </property>
          <item>
           <widget class="QLabel" name="placementXLabel">
            <property name="text">
             <string>X</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="placementXSpinBox">
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>999</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="placementYLabel">
            <property name="text">
             <string>Y</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="placementYSpinBox">
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>999</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="stringsCountSpinBox">
         <property name="visible">
          <bool>false</bool>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>99</number>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="contentLayout">
     <item>
      <layout class="QVBoxLayout" name="pageListLayout">
       <item>
        <layout class="QHBoxLayout" name="pageButtonsLayout">
         <item>
          <widget class="QPushButton" name="pageInsertButton">
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>22</height>
            </size>
           </property>
           <property name="text">
            <string>Insert</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pageDeleteButton">
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>22</height>
            </size>
           </property>
           <property name="text">
            <string>Delete</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListWidget" name="pagesListWidget">
         <property name="minimumSize">
          <size>
           <width>110</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QGroupBox" name="pageDataGroup">
       <property name="minimumSize">
        <size>
         <width>450</width>
         <height>0</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>550</width>
         <height>0</height>
        </size>
       </property>
       <property name="title">
        <string>Page Data</string>
       </property>
       <layout class="QGridLayout" name="pageDataLayout">
        <item row="6" column="0" colspan="2">
         <layout class="QVBoxLayout" name="variationWrapper">
          <item>
           <layout class="QHBoxLayout" name="variationLayout">
            <property name="spacing">
             <number>4</number>
            </property>
            <item>
             <widget class="QCheckBox" name="variationPresentCheckBox">
              <property name="text">
               <string>Variation</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="variationVariableComboBox">
              <property name="minimumSize">
               <size>
                <width>180</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="variationVariablePickerButton">
              <property name="maximumSize">
               <size>
                <width>24</width>
                <height>22</height>
               </size>
              </property>
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="variationEqualsLabel">
              <property name="minimumSize">
               <size>
                <width>20</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>20</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>=</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="variationConstantSpinBox">
              <property name="minimumSize">
               <size>
                <width>60</width>
                <height>0</height>
               </size>
              </property>
              <property name="minimum">
               <number>-9999</number>
              </property>
              <property name="maximum">
               <number>9999</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="variationHintLabel">
            <property name="text">
             <string>â˜†Change common variables before world starts. Example: Use common variable "Difficulty" to separate Normal and Hard modes</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="worldNameLabel">
          <property name="text">
           <string>World Name</string>
          </property>
         </widget>
        </item>
        <item row="11" column="0" colspan="2">
         <widget class="QCheckBox" name="playAfterClearCheckBox">
          <property name="text">
           <string>Playable after clearing</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="worldNameLineEdit"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="eventTypeLabel">
          <property name="text">
           <string>Event Type</string>
          </property>
         </widget>
        </item>
        <item row="7" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="worldNumberLabel">
            <property name="text">
             <string>World Number</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="worldNumberSpinBox">
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>0</height>
             </size>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="maximum">
             <number>99</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="worldNumberHintLabel">
            <property name="text">
             <string>â€»0 means no number (cleared from the start)</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_5">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="0" column="0" colspan="2">
         <widget class="QGroupBox" name="appearanceGroup">
          <property name="title">
           <string>Appearance Condition</string>
          </property>
          <layout class="QGridLayout" name="appearanceLayout">
           <item row="0" column="0" colspan="3">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="spacing">
              <number>12</number>
             </property>
             <item>
              <widget class="QLabel" name="appearanceWorldLabel">
               <property name="text">
                <string>World</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="appearanceWorldSpinBox">
               <property name="minimumSize">
                <size>
                 <width>40</width>
                 <height>0</height>
                </size>
               </property>
               <property name="minimum">
                <number>0</number>
               </property>
               <property name="maximum">
                <number>99</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="appearanceWorldHintLabel">
               <property name="text">
                <string>to clear (0 means always appear)</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item row="3" column="0" colspan="3">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="totalScoreLabel">
               <property name="text">
                <string>Total Score</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="totalScoreSpinBox">
               <property name="minimum">
                <number>0</number>
               </property>
               <property name="maximum">
                <number>999999</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="totalScoreHintLabel">
               <property name="text">
                <string>or above</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Orientation::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item row="1" column="0" colspan="3">
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QComboBox" name="appearanceVariableComboBox">
               <property name="minimumSize">
                <size>
                 <width>120</width>
                 <height>0</height>
                </size>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="appearanceVariableLayout">
               <property name="spacing">
                <number>4</number>
               </property>
               <item>
                <widget class="QPushButton" name="appearanceVariablePickerButton">
                 <property name="maximumSize">
                  <size>
                   <width>24</width>
                   <height>22</height>
                  </size>
                 </property>
                 <property name="text">
                  <string>...</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSpinBox" name="appearanceConstantSpinBox">
                 <property name="minimum">
                  <number>-9999</number>
                 </property>
                 <property name="maximum">
                  <number>9999</number>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="appearanceComparisonComboBox">
                 <property name="minimumContentsLength">
                  <number>1</number>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="startStageLabel">
          <property name="text">
           <string>Start Stage</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="graphicLabel">
          <property name="text">
           <string>Graphic</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <layout class="QHBoxLayout" name="startStageLayout">
          <property name="spacing">
           <number>4</number>
          </property>
          <item>
           <widget class="QLineEdit" name="startStageLineEdit"/>
          </item>
          <item>
           <widget class="QPushButton" name="startStageBrowseButton">
            <property name="text">
             <string>Browse</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="1">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QComboBox" name="eventTypeComboBox">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="12" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="onGameClearLabel">
            <property name="text">
             <string>On Game Clear</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="onGameClearComboBox">
            <property name="minimumSize">
             <size>
              <width>180</width>
              <height>0</height>
             </size>
            </property>
            <property name="baseSize">
             <size>
              <width>180</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_6">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item row="10" column="0" colspan="2">
         <widget class="QCheckBox" name="passWithoutClearCheckBox">
          <property name="text">
           <string>Passable without clearing</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <layout class="QHBoxLayout" name="graphicLayout">
          <property name="spacing">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QComboBox" name="graphicComboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="graphicPickerButton">
            <property name="maximumSize">
             <size>
              <width>24</width>
              <height>22</height>
             </size>
            </property>
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="graphicPreview">
            <property name="minimumSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::Shape::Box</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Shadow::Sunken</enum>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>